cluster-autoscaler/go.mod

```diff
@@ -1,6 +1,6 @@
 module k8s.io/autoscaler/cluster-autoscaler
 
-go 1.21
+go 1.21.6
```

Comment:
Why is the required Go version bumped? It doesn't seem like github.com/google/uuid v1.6.0 requires anything higher than go1.21.

Reply:
@towca thanks, I returned the previous version 1.21